styles/_base.scss

```diff
@@ -146,22 +147,18 @@ ul, ol {
   }
 }
 
-// temporary status bar implementation
-#statusBarContainer {
-  color: black; // use color var when refactoring
-  background-color: rgba(0,0,0,0.3); // use color var when refactoring
-  height: 20px;
-  position: fixed;
-  bottom: 0;
-  left: 0;
-  right: 0;
-  font-size: 16px;  // use rem when refactoring
-  padding: 5px;
-
-  &::after {
-    content: '(temporary Walmart of status bars)';
-    position: absolute;
-    right: 5px;
-    top: 5px;
-  }  
+// turn off animations and transitions with this style
+
+#ov1.minEffects,
```

Comment:
I initially needed this for one of the status bar tests... and then I switched approaches and didn't need it anymore. So, we're not using this CSS as of now, but since I think we will, I left it in. If you don't think we're going to carry over this functionality into v2, let me know and I could pull it out.